### PR TITLE
add Events and Interaction page

### PR DESCRIPTION
**Pull request by SaraVieira**
https://codesandbox.io/s/interaction-98ppy
Page: https://website-git-interactions-pmndrs.vercel.app/react-three-fiber/getting-started/events-and-interaction

### CODE REVIEW

File: docs/react-three-fiber/getting-started/events-and-interaction.mdx [hunk 1]
```diff
@@ -1,7 +1,78 @@
...
+
+We did it! We created the most boring interaction in the story of 3D and we made an alert show up. Now let's make it actually animate our mesh.
+
+Let's start by setting some step to check if the mesh is active:
+
+```jsx
+const [active, setActive] = useState(false)
+```
+
+After we have this we can change the scale with a ternary operator like so:
```
Comment (gsimone) [inline docs/react-three-fiber/getting-started/events-and-interaction.mdx:52]:
```suggestion
After we have this we can set the scale with a ternary operator like so:
```

File: docs/react-three-fiber/getting-started/events-and-interaction.mdx [hunk 2]
```diff
@@ -1,7 +1,78 @@
...
+```jsx
+<mesh onClick={() => alert('Helooo')} ref={myMesh}>
+  <boxBufferGeometry />
+  <meshPhongMaterial color="royalblue" />
+</mesh>
+```
+
+We did it! We created the most boring interaction in the story of 3D and we made an alert show up. Now let's make it actually animate our mesh.
+
+Let's start by setting some step to check if the mesh is active:
```
Comment (gsimone) [inline docs/react-three-fiber/getting-started/events-and-interaction.mdx:46]:
```suggestion
Let's start by setting some state to check if the mesh is active:
```

File: docs/react-three-fiber/getting-started/events-and-interaction.mdx [hunk 3]
```diff
@@ -1,7 +1,78 @@
...
+  onUpdate={(self) => console.log('props have been updated')}
+/>
+```
+
+From this we can see that what we need to do is use the old `onClick` method we use on any DOM element to react to a user clicking the mesh.
+
+Let's add it then:
+
+```jsx
+<mesh onClick={() => alert('Helooo')} ref={myMesh}>
```
Comment (gsimone) [inline docs/react-three-fiber/getting-started/events-and-interaction.mdx:38]:
```suggestion
<mesh onClick={() => alert('Hellooo')} ref={myMesh}>
```

**Review by gsimone — APPROVED** [review]
Only one li'l note, I don't know if I'd call it "animation", since it just changes props, but other than that I love this mini tutorial, good stuff!

File: docs/react-three-fiber/getting-started/events-and-interaction.mdx (hunk 3 above)
Comment (SaraVieira) [reply docs/react-three-fiber/getting-started/events-and-interaction.mdx]: Damn, i misspelled my misspelling

**Comment by SaraVieira** [comment]
You have a point! Will update that :)

Also committed all your requested changes

**Pull request merged**